modules/nw-disabling-hsts.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id="nw-disabling-hsts_{context}"]
+= Disabling HTTP strict transport security per-domain or globally
+
+To disable HTTP strict transport security (HSTS) per-route or globally, you can set the `max-age` value in the route annotation to `0`.
```

Comment:
There's not really the concept of disabling HSTS globally.  You could keep this section for the per-domain description.  Maybe that should be "per route" rather than "per domain".